stats/view/worker.go

```diff
@@ -50,8 +51,7 @@ var defaultWorker *worker
 
 var defaultReportingDuration = 10 * time.Second
 
-// Find returns a registered view associated with this name.
-// If no registered view is found, nil is returned.
+// Deprecated: No replacement.
```

Comment:
Find is useful if you need a pointer to an existing View by name and unsubscribe from it. We shouldn't remove it.